c_src/eleveldb.cc

```diff
@@ -245,6 +253,8 @@ class eleveldb_priv_data
     EleveldbOptions m_Opts;
     leveldb::HotThreadPool thread_pool;
     leveldb::HotThreadPool stream_thread_pool;
+    leveldb::ExpiryPtr_t m_ExpiryModule;    // only populated if expiry options seen
```

Comment:
m_ExpiryModule is not present in final code version.  Dead code ... simply erase this line and next.